src/Message/CaptureRequest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace MyOnlineStore\Omnipay\KlarnaCheckout\Message;
+
+use Klarna\Rest\OrderManagement\Order;
+
+final class CaptureRequest extends AbstractRequest
+{
+    use ItemDataTrait;
+
+    /**
+     * @inheritDoc
+     */
+    public function getData()
+    {
+        $this->validate('transactionReference', 'amount');
```

Comment:
I could however add `order_lines` to the output conditionally, instead of passing an empty array